js/news-article.js

```diff
@@ -133,3 +135,27 @@ function renderNews() {
 
     }
 }
+
+function sharefacebook() {
+    shareOverrideOGMeta(newsTitle, newsImageSource);
+};
+function shareOverrideOGMeta(overrideTitle, overrideImage) {
+    var params = (new URL(document.location)).searchParams;
+    var newsTitleEnc = encodeURI(params.get("title"));

```

Comment:
Use NewsTitle variable

Reply:
newsTitle variable may not give correct results